test/extended/networking/network_segmentation.go

```diff
@@ -793,7 +789,7 @@ func generateCIDRforUDN(oc *exutil.CLI) string {
 `
 	} else if hasIPv6 {
 		cidr = `
-   - cidr: 2014:100:200::0/60
+    - cidr: 2014:100:200::0/60
```

Comment:
This change seem to belong previous commit.

Reply:
Are you sure? This is a port of https://github.com/ovn-org/ovn-kubernetes/commit/336fb4457c5fa038468b8a10e0a52e33a68362e3 while the preview commit is a port of https://github.com/ovn-org/ovn-kubernetes/pull/4486/commits/bef4a75b5d95f1db884deedeb109910546eb42bf